examples/darwin-framework-tool/commands/tests/TestCommandBridge.h

```diff
@@ -154,6 +154,9 @@ class TestCommandBridge : public CHIPCommandBridge,
         VerifyOrReturnError(controller != nil, CHIP_ERROR_INCORRECT_STATE);
 
         SetIdentity(identity);
+        if (controller.controllerNodeId != nil) {
+            mCommissionerNodeId.SetValue([controller.controllerNodeId unsignedLongLongValue]);
```

Comment:
This part worries me: where are we setting mCommissionerNodeId for "normal" YAML steps?  It should be getting set from the identity for each step, and I don't see that happening in this PR, at least for darwin-framework-tool.

Reply:
One of pair or WaitForCommissionee needs to be called before you can run any "normal" yaml step. This will set the value for mCommissionerNodeId in the base class. It follows the places where "SetIdentity" is used (though I'm not sure that's actually used for anything)

I think the issue might be in multi-admin scenarios where you use an identity: to set the commissioner on the fly and then also use the commissionerNodeId variable. 

I can set the node id in GetDevice. Should probably call SetIdentity there too. I think that would cover this scenario. Will make the change and test (need a day or two to get caught back up with everything).

Reply:
kk, fixed this up and added tests for using different identities and also for using the node id in the response and arguments.